sdk/azidentity/client_certificate_credential_test.go

```diff
@@ -224,6 +225,53 @@ func TestClientCertificateCredential_Live(t *testing.T) {
 			testGetTokenSuccess(t, cred)
 		})
 	}
+	t.Run("disableID", func(t *testing.T) {
```

Comment:
Consistent with other tests, avoids further overloading "ID"
```suggestion
	t.Run("instance discovery disabled", func(t *testing.T) {
```